stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -100,102 +107,180 @@ public Stripe(String publishableKey) throws AuthenticationException {
     }
 
     /**
-     * Call to create a {@link Token}.
+     * The simplest way to create a token, using a {@link Card} and {@link TokenCallback}. This
+     * runs on the default {@link Executor} and with the
+     * currently set {@link #defaultPublishableKey}.
+     *
+     * @param card the {@link Card} used to create this payment token
+     * @param callback a {@link TokenCallback} to receive either the token or an error
+     */
+    public void createToken(@NonNull final Card card, @NonNull final TokenCallback callback) {
+        createToken(card, defaultPublishableKey, callback);
+    }
+
+    /**
+     * Call to create a {@link Token} with a specific public key.
      *
      * @param card the {@link Card} used for this transaction
      * @param publishableKey the public key used for this transaction
      * @param callback a {@link TokenCallback} to receive the result of this operation
      */
     public void createToken(
-            final Card card,
-            final String publishableKey,
-            final TokenCallback callback) {
+            @NonNull final Card card,
+            @NonNull final String publishableKey,
+            @NonNull final TokenCallback callback) {
         createToken(card, publishableKey, null, callback);
     }
 
-    public void setDefaultPublishableKey(String publishableKey) throws AuthenticationException {
-        validateKey(publishableKey);
-        this.defaultPublishableKey = publishableKey;
+    /**
```

Comment:
The reason the methods are showing up in green instead of just the Javadoc is that I moved the methods around in the file to create a hierarchy of public > package > private